LondonBritishLibrary/orient/BLorient12860.xml

```diff
@@ -0,0 +1,207 @@
+                            </decoNote>
+                        </decoDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc type="Correction">The text has been restored by a European hand in pencil either between the lines or in the
+                                        upper margin throughout the manuscript. The corrections seem to give the results of collating the text with
+                                        <bibl><ptr target="bm:ContiRossini1904YaredPantalewonT"/></bibl>.</desc> 
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#16ra"/>
+                                    <desc type="OwnershipNote">The place left for the name of the owner has not been filled in (<locus target="#16ra"/> 
+                                        and other places).</desc>
+                                </item>
+                                <item xml:id="e3">
+                                    <locus target="#17r"/>
+                                    <desc type="OwnershipNote">In the lower margin a signature in Cyrillic 
+                                        (<persName ref="PRS14477Pechaguev" role="owner">Т. Печагуев</persName>) is repeated four times.</desc>
+                                </item>
+                                <item xml:id="e4">
+                                    <locus target="#9r #21r #Ir"/>
+                                    <desc type="StampExlibris">The round stamp of 
+                                        <persName ref="PRS14476Bernard" role="owner">Marius Bernardus Angelus Comnenus</persName> 
+                                        with the arms of his family in the centre.</desc>
+                                </item>
+                                <item xml:id="e5">
+                                    <locus target="#Ir"/>
+                                    <desc type="StampExlibris">Ex libris with inscription.</desc>
+                                    <q xml:lang="fr">Maison Impériale du Prince Marius Bernard Angelo de Thessalie</q>
+                                </item>
+                                <item xml:id="e6">
+                                    <locus target="#Ir"/>
+                                    <desc type="StampExlibris">A rectangular stamp.</desc>
+                                    <q xml:lang="fr">Archives de la Maison Impériale / Angelo Comneno de Thessalie</q>
+                                </item>
+                                <item xml:id="e7">
+                                    <locus target="#Ir"/>
+                                    <desc type="AcquisitionNote">Note in ink.</desc>
+                                    <q xml:lang="fr">Evangile en parchemin originel de XVIIèm siècle en langue gheez donné à moi par un prêtre en
```

Comment:
There are some orthographic peculiarities in the French that Strelcyn marked with "sic". I would take over the "sic"s that he has given.